src/python/tensorflow_cloud/core/tests/unit/containerize_test.py

```diff
@@ -47,7 +48,7 @@ def assert_docker_file(self, expected_lines, docker_file):
             actual_lines = f.readlines()
             self.assertListEqual(expected_lines, actual_lines)
 
-    def test_create_docker_file_defaults(self):
+    def test_create_docker_file_defaults(self, MockRequests):
```

Comment:
do you not have to mock return value to be `ok == True`?

Reply:
Or, otherwise, since this argument is actually unused in the test method, it should be named something like `unused_mock_requests`

(btw; you seem to be using snake_case for variable names. You should be consistent to be mock_requests.

Reply:
Setting it to True is not required but added it anyway for readability and renamed all mock variables.